src/codec_aom.c

```diff
@@ -689,6 +696,21 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         cfg->g_input_bit_depth = image->depth;
         cfg->g_w = image->width;
         cfg->g_h = image->height;
+
+        // Detect the libaom v3.6.0 bug described in
+        // https://crbug.com/aomedia/2871#c12. See the changes to
+        // av1/encoder/encoder.c in
+        // https://aomedia-review.googlesource.com/c/aom/+/174421.
+        static const int aomVersion_3_6_0 = (3 << 16) | (6 << 8);
```

Comment:
This is OK. For your other option, were you thinking of failing at configure time or failing at runtime for all uses of 3.6.0?

Reply:
I was thinking of sending email to the relevant mailing lists about this serious bug in libaom 3.6.0. I wasn't thinking of failing at configure time or failing at runtime for all uses of 3.6.0.